README.md

```diff
@@ -0,0 +1,38 @@
+# eslint-config-lintification
+
+A suite of battle-tested linting rules for any modern Node.js application, with full support for the most common TypeScript rules.
+
+
+
+## Table of Contents
+- [eslint-config-lintification](#eslint-config-lintification)
+  - [Table of Contents](#table-of-contents)
+  - [Usage](#usage)
+    - [Install](#install)
+    - [Extend the Configuration](#extend-the-configuration)
+  - [License](#license)
```

Comment:
I included the table of contents out of habit. But since it's such a short README, it's probably not necessary.
```suggestion
```